app/src/main/java/vn/tien/photo_world/adapters/PhotoAdapter.java

```diff
@@ -1,4 +1,82 @@
 package vn.tien.photo_world.adapters;
 
-public class PhotoAdapter {
+import android.content.Context;
+import android.content.Intent;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.bumptech.glide.Glide;
+
+import java.util.List;
+
+import vn.tien.photo_world.R;
+import vn.tien.photo_world.data.model.Photo;
+import vn.tien.photo_world.ui.activity.EditActivity;
+
+public class PhotoAdapter extends RecyclerView.Adapter<PhotoAdapter.ViewHolder> {
+    private List<Photo> mPhotoList;
+    private Context mConText;
+
+    public PhotoAdapter(List<Photo> photoList, Context context) {
```

Comment:
```suggestion
    public PhotoAdapter(Context context, List<Photo> photos) {
```
context luôn là tham số đầu tiên